clippy_lints/src/utils/internal_lints.rs

```diff
@@ -337,6 +337,15 @@ declare_clippy_lint! {
     "found clippy lint without `clippy::version` attribute"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Check that the `extract_msrv_attr!` macro is used, when a lint has a MSRV.
+    ///
+    pub INVALID_MSRV_ATTR_IMPL,
```

Comment:
Nice to see an implementation this quickly after the suggestion! I only skimmed over the code, maybe `MISSING_MSRV_ATTR_IMPL` would be a slightly better name :thinking:. :upside_down_face:

Reply:
@xFrednet feel free to self-assign this PR if you like

Reply:
I'm still trying to limit my work on Clippy while writing my bachelor thesis. Clippy is just more fun and productive :joy:. I have some other reviews waiting on me, if those are further along, I can take another look at it. But that might take some time, so for a timely review, it's probably better if you stay assigned. But thank you very much for the offer! :)

Reply:
Yeah I was unsure about the name.

(I intentionally assigned Cam, so that you don't get distracted again)

@camsteffen Since you collected a lot of reviews recently, you can leave this PR at the back of your review queue. There shouldn't be any rush getting this in.